### ISSUE TITLE
"_" is not exported by swc

### ISSUE THREAD

**Issue by linonetwo** [wessberg/rollup-plugin-ts#211]
- Version:
- Rollup Version:
- Operating System and version (if applicable):
- Node Version (if applicable):
- Does it work with `tsc` (if applicable):

### Reproduction

```
import type { JSONSchema } from 'json-schema-typed';

export class ValidationError extends Error {
  key: string;
  value: unknown;
  schema: JSONSchema;
  nodeInfo: string;
  constructor(key: string, value: unknown, schema: JSONSchema, nodeInfo: string) {
    super();
    this.name = 'ValidationError';
    this.message = `Value ${key} does not meet data requirement:\n${JSON.stringify(schema)}\n${JSON.stringify(value)}`;
    this.key = key;
    this.value = value;
    this.schema = schema;
    this.nodeInfo = nodeInfo;
  }
}
```

```
import type { Options as SwcOptions } from '@swc/core';
import fs from 'fs';
import ts from 'rollup-plugin-ts';

// const swcrc = JSON.parse(fs.readFileSync('../../.swcrc', 'utf8')) as SwcOptions;

export default {
  input: 'src/index.ts',
  output: {
    dir: 'dist',
    format: 'es',
  },
  plugins: [
    ts({
      transpiler: 'swc',
      browserslist: ['last 1 version', '> 1%'],
      swcConfig: {
        // ...swcrc,
      },
    }),
  ],
};
```

### Expected Behavior

Don't know why swc do this.

### Actual Behavior

```
[!] RollupError: "_" is not exported by "../../node_modules/.pnpm/@swc+helpers@0.5.0/node_modules/@swc/helpers/cjs/_inherits.cjs", imported by "src/error/index.ts".
https://rollupjs.org/troubleshooting/#error-name-is-not-exported-by-module
src/error/index.ts (4:9)
2: import { _ as _class_call_check } from "@swc/helpers/_/_class_call_check";
3: import { _ as _define_property } from "@swc/helpers/_/_define_property";
4: import { _ as _inherits } from "@swc/helpers/_/_inherits";
            ^
5: import { _ as _wrap_native_super } from "@swc/helpers/_/_wrap_native_super";
6: import { _ as _create_super } from "@swc/helpers/_/_create_super";
```

**Comment by linonetwo** [comment]
This works without swc, but not generating .d.ts file, Why?

```
import ts from 'rollup-plugin-ts';

// const swcrc = JSON.parse(fs.readFileSync('../../.swcrc', 'utf8')) as SwcOptions;

export default {
  input: 'src/error/index.ts',
  output: {
    dir: 'dist',
    format: 'es',
  },
  plugins: [
    ts({
      swcConfig: {
        // ...swcrc,
      },
    }),
  ],
};

```

**Comment by wessberg** [comment]
Hi there,

This should be resolved as part of [v3.3.0](https://github.com/wessberg/rollup-plugin-ts/releases/tag/v3.3.0), so I'm closing this issue.
